app/assets/stylesheets/global.css.scss

```diff
@@ -0,0 +1,125 @@
+$dark_gray: #2B2E2E;
+$light_gray: #C0C1C1;
+$green: #33CD99;
+
+@import url(http://fonts.googleapis.com/css?family=Amaranth:700italic);
+
+@mixin bold_font {
+  font-family: "Brandon Grotesque","Helvetica Neue", Helvetica, sans-serif;
+  font-weight: bold;
+  text-transform: uppercase;
+}
+
+* {
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+}
+
+body {
+  padding-top: 0rem;
+  font-family: "Open Sans", "Helvetica Neue", Helvetica, sans-serif;
+}
+
+.wrapper {
+  width: 80%;
+  margin: 0 auto;
+}
+
+.wrapper_with_padding {
+  width: 80%;
+  margin: 0 auto;
+  padding: 3.5rem 0;
+}
+
+.clearfix:before, .clearfix:after {
+  content: " ";
+  display: table;
+}
+
+.clearfix:after {
+  clear: both;
+}
+
+button, .button {
+  border: none;
+  outline: none;
+  background: $green;
+  padding: 1rem 3.5rem;
+  border-radius: 2rem;
+  a {
+    color: white;
+    text-decoration: none;
+  }
+}
+
+.button {
+  color: white;
+  margin: 2rem 0;
+}
+
+a {
+  color: $green;
+  text-decoration: none;
+  &:hover {
+    color: $dark_gray;
+  }
+}
+
+header {
+  background: $dark_gray;
+  position: fixed;
```

Comment:
Properties should be ordered background, border-bottom, padding, position, top, width